src/components/GenreDropdown/GenreDropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Fragment } from "react";
+
+export class GenreDropdown extends React.PureComponent<any, any> {
+  options = [
+    { label: "Select genre", value: "" },
+    { label: "Crime", value: "Crime" },
+    { label: "Documentary", value: "Documentary" },
+    { label: "Horror", value: "Horror" },
+    { label: "Comedy", value: "Comedy" },
+  ];
+  constructor(props) {
+    super(props);
+    this.state = { options: this.options, value: "" };
+  }
+
+  handleChange = (event) => {
+    this.setState({ value: event.target.value });
+  };
+
+  render() {
+    const { options, value } = this.state;
+
+    return (
+      <Fragment>
```

Comment:
`<Fragment>` can be replaced with just `<>`

Reply:
thank you didn't know about that